### PR TITLE
don't quote None

### PR DESCRIPTION
**Pull request by psegedy**

### CODE REVIEW

File: src/widgetastic/xpath.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 def quote(s):
     """Quotes a string in such a way that it is usable inside XPath expressions."""
+    if not s:
```
Comment (RonnyPfannschmidt) [inline src/widgetastic/xpath.py:8]: should this act different for a empty string?

Comment (RonnyPfannschmidt) [reply src/widgetastic/xpath.py:8]:
```suggestion
    if s is None:
```

Comment (psegedy) [reply src/widgetastic/xpath.py]: empty string should be quoted as empty string, but explicit comparison is clearer

**Comment by quarckster** [comment]
Closed in favor #207

**Pull request closed without merging**